fe/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java

```diff
@@ -842,11 +843,24 @@ private void createJob(long dbId, OlapTable olapTable, Map<Long, LinkedList<Colu
         // property 3: timeout
         long timeoutSecond = PropertyAnalyzer.analyzeTimeout(propertyMap, Config.alter_table_timeout_second);
 
+        // Parse storage_format property
+        // sql: alter table tablet_name set ("storage_format" = "v2")
+        // Use this sql to convert all tablets(base and rollup index) to segment v2.
+        // analyzeStorageFormat will parse the properties to decide whether storage format is set to v2.
+        // If true, fire a schema change job to achieve it.
+        boolean changeStorageFormat = PropertyAnalyzer.analyzeStorageFormat(propertyMap);
```

Comment:
If the storage format of this table is already v2, will another schema change job be created?
It's not suitable to return a boolean value, a format type is perferred, and the caller will process it as it wants. For example if there is another storage format in the future, you will find that the analyzeStorageFormat should be changed, but it may make other user unhappy.

Reply:
> If the storage format of this table is already v2, will another schema change job be created?

This function will only be used to force to convert a v1 table(maybe partially is v1) to v2 manually. Should use it before check the table. It is for simplicity.
And more, Because In FE, we do not store the storage format information of table, So it is hard to reject to convert a v2 table.

Reply:
Can it be checked in backend to avoid unnecessary operation?